File: src/pagePackage/ApprovedCoursesPage.java
package pagePackage;

import systemMessagePackage.FunctionType;
import systemMessagePackage.SystemMessage;

public class ApprovedCoursesPage extends Page{

	public ApprovedCoursesPage(String content) {
		super(content);
		setType(PageType.APPROVED_COURSES_PAGE);
		setName("Approved Courses Page");
	}
	
	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE_STUDENT, null);
	}
}

File: python_development/Page/ApprovedCoursesPage.py
from python_development.Page.Page import Page
from python_development.Page.PageType import PageType
from python_development.System.FunctionType import FunctionType
from python_development.System.SystemMessage import SystemMessage

class ApprovedCoursesPage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType("APPROVED_COURSES_PAGE")
        self.setName("Approved Courses Page")

    
    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, None)